leptonai/photon/tests/test_photon.py

```diff
@@ -1309,6 +1309,7 @@ def fast(self):
         client = Client(f"http://127.0.0.1:{port}")
         for i in range(3):
             self.assertEqual(client.fast(), "fast")
+            time.sleep(0.001)
```

Comment:
Note: this is to make sure that the request QPS is going to be mathematically under the threshold. Before this we are at the mercy of not-so-fast-cpus.